redbeams/src/main/java/com/sequenceiq/redbeams/service/network/NetworkParameterAdder.java

```diff
@@ -21,10 +21,9 @@
         return parameters;
     }
 
-    public Map<String, Object> addVpcParameters(Map<String, Object> parameters, EnvironmentNetworkResponse environmentNetworkResponse) {
-        parameters.put(VPC_ID, String.join(",", environmentNetworkResponse.getAws().getVpcId()));
-        parameters.put(VPC_CIDR, environmentNetworkResponse.getNetworkCidr());
+    public Map<String, Object> addVpcParameters(Map<String, Object> parameters, DetailedEnvironmentResponse environmentResponse) {
+        parameters.put(VPC_CIDR, environmentResponse.getSecurityAccess().getCidr());
+        parameters.put(VPC_ID, environmentResponse.getNetwork().getAws().getVpcId());
```

Comment:
I am surprised to see getAws() in a class and package that does not seem to be AWS-specific.

Reply:
Yep, you are right. So I cured that place.